examples/chart/teleport-cluster/values.yaml

```diff
@@ -27,6 +27,16 @@ authenticationType: local
 # Possible values are 'multiplex'
 proxyListenerMode: ""
 
+# Optional setting for configuring session recording. Possible values are:
+#    "node"  : sessions will be recorded on the node level  (the default)
+#    "node-sync" : session recordings will be streamed from node -> auth -> storage service without being stored on disk at all.
```

Comment:
```suggestion
#    "node-sync" : session recordings will be streamed from Node -> Auth Service -> storage service without being stored on disk at all.
```
To capitalize product names

Reply:
This is copied from https://github.com/gravitational/teleport/blob/master/docs/pages/setup/reference/config.mdx